docs/sources/features/explore/index.md

```diff
@@ -84,24 +84,31 @@ Along with metrics, Explore allows you to investigate your logs with the followi
 - [InfluxDB](../datasources/influxdb)
 - [Elasticsearch](../datasources/elasticsearch)
 
-### Deduping
+### Visualization Options
+
+#### Deduping
 
 Log data can be very repetitive and Explore can help by hiding duplicate log lines. There are a few different deduplication algorithms that you can use:
 
 * `exact` Exact matches are done on the whole line, except for date fields.
 * `numbers` Matches on the line after stripping out numbers (durations, IP addresses etc.).
 * `signature` The most aggressive deduping - strips all letters and numbers, and matches on the remaining whitespace and punctuation.
 
-### Timestamp and Local time
+#### Time
+
+Shows or hides the time column. This is the timestamp associated with the log line as reported from the datasource.
```

Comment:
Per the Documentation Style Guide, "data source" is two words.

Reply:
✅ updated